python/altrios/tests/test_param_paths.py

```diff
@@ -0,0 +1,107 @@
+import unittest
+import altrios as alt
+from .mock_resources import *
+
+class TestParamPath(unittest.TestCase):
+    def test_param_path_list(self):
+        from altrios import LocomotiveSimulation, Locomotive, PowerTrace
+        mock_df = mock_pymoo_conv_cal_df()
+        pt = PowerTrace(
+            mock_df['time [s]'],
+            mock_df['Tractive Power [W]'],
+            engine_on=mock_df['engine_on'],
+        )
+        loco_sim = LocomotiveSimulation(
+            loco_unit=Locomotive.default(),
+            power_trace=pt,
+            save_interval=1)
+
+        baseline_variable_paths = ['power_trace.time_seconds', 'power_trace.pwr_watts', 'power_trace.time_hours', 'power_trace.engine_on', 'loco_unit.edrv.save_interval', 
```

Comment:
I think I'd like to see this list in a separate file (not sure where pathwise but; see if you can find a good location) and loaded by the script using e.g. 
```python
with Open(fsim.resources_dir() / "test_assets/baseline_variable_paths.txt"
```
where the file is a plain text file with 1 string per line.